nagios/stackstate_checks/nagios/nagios.py

```diff
@@ -458,3 +433,32 @@ def _get_metric_prefix(self, line_data):
 
 class InvalidDataTemplate(Exception):
     pass
+
+
+def create_event(timestamp, event_type, hostname, fields):
+    """
+    Factory method called by the parsers
+    """
+    event = fields._asdict()
+    tags = ['{}:{}'.format(k, v) for k, v in event.items()]
+
+    # if host is localhost, turn that into the internal host name
```

Comment:
any reason why you moved this above ?
before your change it was after the `event.update`

Reply:
Good that you spotted that, no need for this change. 
I was testing one other approach in StackPack so I was sending a bit more data with tags. 
By mistake I didn't move the host part to its original line when I reverted the code.